cvmfs/swissknife_main.cc

```diff
@@ -193,6 +195,32 @@ int main(int argc, char **argv) {
              ->PrintList(perf::Statistics::kPrintHeader).c_str());
   }
 
+  if (command->GetName() == "sync" || command->GetName() == "ingest") {
+    StatisticsDatabase *db;
```

Comment:
This can be a `UniquePtr<StatisticsDatabase>` which doesn't need a manual `delete`.